lib/hanami/view/erb/engine.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "temple"
+require_relative "parser"
+require_relative "filters/block"
```

Comment:
@timriley Nitpick: please move these `require_relative` inside the module definition.

Ruby Reflection is more straightforward, so when we do `Hanami::View.const_location(:ERB)`, we get this file instead of `filters/block.rb`.

Reply:
Done in 6eb4637

> Ruby Reflection is more straightforward, so when we do Hanami::View.const_location(:ERB), we get this file instead of filters/block.rb.

Thanks for sharing this particular detail; I've never thought of it along these lines.